src/main/haskell/kore/src/Kore/Step/BaseStep.hs

```diff
@@ -204,11 +234,12 @@ stepWithAxiom
     -> ExceptT
         (Simplifier (StepError level variable))
         Simplifier
-        (ExpandedPattern.ExpandedPattern level variable
+            ( ExpandedPattern.ExpandedPattern level variable
```

Comment:
I think that you should have unindented the next two lines instead of indenting this one, since this parenthesis is a direct child of ExceptT.